src/include/cell.i

```diff
@@ -936,7 +936,11 @@ __wt_cell_unpack_dsk(
      * Previous startup txnid=0, ts=y       txnid=0, ts=1           txnid=MAX, ts=MAX
      */
     if (dsk->write_gen <= S2C(session)->base_write_gen) {
-        unpack->start_txn = WT_TXN_NONE;
+        /* Let reconciliation know we clear the transaction ids and the cell needs to be rebuilt. */
+        if (unpack->start_txn != WT_TXN_NONE) {
+            unpack->start_txn = WT_TXN_NONE;
+            F_SET(unpack, WT_UNPACK_TXNID_CLEARED);
+        }
```

Comment:
Because of where the `WT_UNPACK_TXNID_CLEARED` flag is set, doesn't it mean "we cleared ONLY the start transaction ID", not  "we cleared the transaction ids"?

Reply:
If we cleared the start transaction id and we have a stop transaction id, the stop transaction id must have been cleared as well.

Reply:
In which case maybe the name should change?

Reply:
Sure, will rename to WT_UNPACK_TXNIDS_CLEARED.